README.md

```diff
@@ -41,10 +41,12 @@ Sublime Text 3 plugin for generating a Table of Contents (TOC) in a Markdown doc
     - [Customizable list bullets in TOC](#customizable-list-bullets-in-toc)
     - [Specify custom indentation prefix](#specify-custom-indentation-prefix)
     - [Preserve images in headings](#preserve-images-in-headings)
+    - [Excluded heading](#excluded-heading)
```

Comment:
Rewrite to "Excluded headings", since we are describing the feature in generically